modules/testing-with-model-control-protocol-servers.adoc

```diff
@@ -0,0 +1,41 @@
+:_module-type: PROCEDURE
+
+[id="testing-with-model-control-protocol-servers_{context}"]
+= Testing with model control protocol (MCP) servers
+
+[role="_abstract"]
+
+Authorize and interact with connected MCP servers to use their integrated tools directly from the playground chat.
+
+.Prerequisites
+
+* You have configured a playground instance for your project.
+* A cluster administrator has configured an MCP server, and the server is listed and available in the *MCP servers* section of your playground.
```

Comment:
For MCP Tool calling, the user needs to have a model that has tool calling capabilities turned on, otherwise tools wont be called.

Also, larger models are preferred over smaller ones, as tool calling is complex.